README.md

```diff
@@ -236,6 +236,14 @@ or IntelⓇ FPGA RTE for OpenCL™ installed on your system:
 
 6.  Run your OpenCL host program.
 
+
+### Debug Environment Variables
+
+| Environment Variable Title  | Description |
+| ------------- | ------------- |
```

Comment:
Can you please add a description for `ACL_DEBUG` environment variable? It's just a general runtime debug env var, basically output everything else other than `ACL_HAL_DEBUG` and `ENV_ACL_CONTEXT_CALLBACK_DEBUG`. You can search for the code for a brief use case.

Reply:
Added ACL_DEBUG, please look it over when you can :)